app/vite.config.js

```diff
@@ -18,17 +65,35 @@ export default defineConfig({
       input: Object.fromEntries(
         glob
           .sync(path.join(__dirname, "/**/*.html"))
-          .filter((htmlFilePath) => !htmlFilePath.includes("dist/"))
+          .filter(
+            (htmlFilePath) =>
+              !htmlFilePath.includes("dist/") &&
+              !htmlFilePath.includes("tempHtmlFiles/"),
+          )
           .map((htmlFilePath) => {
             const baseName = path.basename(htmlFilePath);
+
+            // Copy over all html files to ./tempHtmlFiles
+            const htmlFileAppPath = htmlFilePath.split("/app/")[1];
+            const tempProdHtmlFilePath = path.resolve(
+              __dirname,
+              `${TEMP_PROD_HTML_DIR}/${htmlFileAppPath}`,
+            );
+            const tempProdHtmlFileDir = path.dirname(tempProdHtmlFilePath);
+            if (!existsSync(tempProdHtmlFileDir)) {
+              mkdirSync(tempProdHtmlFileDir, { recursive: true });
+            }
+            copyFileSync(htmlFilePath, `${tempProdHtmlFileDir}/${baseName}`);
```

Comment:
`pnpm dev` now also creates the `tempHtmlFiles` directory too. Think we
- add `tempHtmlFiles` to `.gitignore`
- have different inputs depending on whether its dev or prod:
```
/**
 *
 * @param {"build" | "serve"} command
 * @returns {{ [entryAlias: string]: string }} //
 */
function htmlInputs(command) {
  const htmlFiles = glob
    .sync(path.join(__dirname, "/**/*.html"))
    .filter(
      (htmlFilePath) =>
        !htmlFilePath.includes("dist/") &&
        !htmlFilePath.includes("tempHtmlFiles/"),
    );
    .... if command === "serve" return the og else do the tempHtmlFiles thing
}
```

Reply:
good catch, updated!